lib/src/clojure_lsp/crawler.clj

```diff
@@ -52,9 +52,7 @@
                                       (producer/publish-progress producer percentage "Analyzing external classpath" progress-token)))
             normalization-config {:external? true
                                   :filter-analysis (fn [analysis]
-                                                     (-> analysis
-                                                         (dissoc :namespace-usages)
```

Comment:
This is the change that necessitates to bump to db/version.

Reply:
Just a note for ourselves related to what was mentioned [here](https://github.com/clojure-lsp/clojure-lsp/pull/1053#pullrequestreview-1008373323):
if we agree on deploying a code that won't affect existing users, we could check the setting here as well to know if we should dissoc

Reply:
I'd vote for consistency, I could see namespace-usages being useful beyond the dep graph.